deployers/servicedeployer.go

```diff
@@ -808,10 +810,17 @@ func (deployer *ServiceDeployer) DeployRules() error {
 
 // Deploy Apis into OpenWhisk
 func (deployer *ServiceDeployer) DeployApis() error {
-	for _, api := range deployer.Deployment.Apis {
-		err := deployer.createApi(api)
-		if err != nil {
-			return err
+	var err error
+	// Only deploy either swagger or manifest defined api, but not both
+	// Swagger API takes precedence
```

Comment:
please add caps NOTE saying the same, Swagger API take precedence over the apis defined in manifest YAML.